fineract-provider/src/main/java/org/apache/fineract/portfolio/charge/service/ChargeReadPlatformServiceImpl.java

```diff
@@ -225,6 +225,7 @@ private void processChargeExclusionsForLoans(ChargeTimeType[] excludeChargeTimes
                 sb.append(excludeChargeTimes[i].getValue());
             }
             excludeClause = excludeClause.append(" and c.charge_time_enum not in(" + sb.toString() + ") ");
+            excludeClause.append(" ");
```

Comment:
I'm guessing you had a good reason to add this, but can I ask why? Did SpotBugs suggest this? Why?

Reply:
excludeClause is never used in this method but gets assigned a value here which is been used by another method. So to please spotbugs, I added the redundant line at the end.